arches_component_lab/src/arches_component_lab/widgets/ResourceInstanceMultiSelectWidget/components/ResourceInstanceMultiSelectWidgetEditor.vue

```diff
@@ -31,9 +31,9 @@ const { $gettext } = useGettext();
 
 const itemSize = 36; // in future iteration this should be declared in the CardXNodeXWidget config
 
-const options = ref<ResourceInstanceReference[]>(props.initialValue || []);
+const options = ref<ResourceInstanceReference[]>([]);
 const isLoading = ref(false);
-const resourceResultsPage = ref(0);
+const resourceResultsPage = ref(1);
```

Comment:
could this be `ref(0)` and still function?

Reply:
Yeah, either way works.